### PR TITLE
document list view

### PR DESCRIPTION
**Pull request by nishlashakya**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ app.use(
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
 app.use(express.static(path.join(__dirname, "utils")));
+// app.use(express.static(path.join(__dirname, "../zegal-client")));
```
Comment (Anthony-Michel) [inline app.js]: commented code

File: routes/index.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
 var express = require("express");
 var router = express.Router();
+const DocumentService = require("../services/document/document.service");
 
 /* GET home page. */
 router.get("/", async function(req, res, next) {
+  const documents = await DocumentService.getAllDocuments();
```
Comment (Anthony-Michel) [inline routes/index.js:7]: what does that do ?
All documents of a specific key user ?

Comment (nishlashakya) [reply routes/index.js:7]: yes

File: app.js (hunk 1 above)
Comment (nishlashakya) [reply app.js:39]: can we keep that here? I need it while using a local build file

Comment (nishlashakya) [reply app.js]: removed we will keep these in config

**Review by Anthony-Michel — APPROVED** [review]

**Pull request merged**